dxos/dxos
# chore: sentry improvements

- bump to latest sentry sdk
- add sentry vite plugin which should notify sentry about releases & upload source maps
- hook up tracing & replay
- add hidden error trigger into halo-app for testing purposes
